AndreaRene/tarot-application
# Edit seed logic to allow multiple deck creation

## Description
This pull request updates the seed logic, modifies the deck seed object structure, and renames the original card seed folder to RWSCardSeeds.json to support multiple decks. This change was necessary to allow the application to handle multiple tarot decks, each with its own set of cards, and to streamline the process of adding new decks in the future.

## Changes Made
- Updated seed logic to support multiple decks. The new logic iterates over each deck in the deck seeds file and dynamically loads the corresponding card seeds file for that deck.
- Modified deck seed object structure to include references to card seed files. Each deck seed object now contains a reference to the filename of the card seeds file for that deck.
- Renamed the original card seed folder to RWSCardSeeds.json to prepare for adding more decks. This naming convention allows for easy identification and addition of new decks.

## Testing
- Run the seeding script with the updated logic for multiple decks.
- Verify that current deck is successfully seeded with respective cards.

## Aditional Notes
We do not currently have multiple decks in the seeds, so further testing will have to wait until they are implemented.
